runtime/integration-test/src/lib.rs

```diff
@@ -11,3 +11,6 @@ mod bootstrap;
 
 #[cfg(any(feature = "with-kusama-runtime",))]
 mod proxy;
+
+#[cfg(any(feature = "with-kusama-runtime",))]
```

Comment:
recently we switched to building both envs at once (kusama & rococo) so maybe we can get rid of conditional compilation ? :thinking:

Reply:
this is different, the feature flag is used to define which runtime to use in the tests
it is because of the imports mostly
in setup we use `with-some-runtime` to import from specific runtimes
or the fees use a bit different values when it comes to rococo vs kusama

we only do tests for kusama now, but we also should for rococo, and eventually polkadot